.github/PULL_REQUEST_TEMPLATE/security_related_pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+---
+name: "Security-related Pull Request"
+about: "This pull request template is intended to be used when there is a change in our attack surface."
+title: "Security-related Pull Request"
+labels: "security"
+assignees: "ronaldheft-usds", "td-usds", "jimduff-usds", "RickHawesUSDS"
+---
+
+
+This PR contains changes which require special attention to security-related aspects
+
+
+## Changes
+-
+-
+
+## Specific Security-related subjects a reviewer should pay specific attention to
+
+- Newly introduced enpoints
+    - new endpoint A
+    - new endpoint B
+- What are the potential security threats and mitigations? Please list the STRIDE threats and how they are mitigated
+    - **S**poofing
+        - Threat T, which could be achieved by A, is mitigated by M
+    - **T**ampering
+    - **R**epudiation
+    - **I**nformation disclosure
+    - **D**enial of service
+    - **E**levation of Privilege
```

Comment:
Nice! Good suggestion to get PR authors to think about their security attack surfaces.